pkg/kube/kube.go

```diff
@@ -149,6 +150,18 @@ func (k *KubeClient) GetApplication(name string) (*wego.Application, error) {
 	return &a, nil
 }
 
+func (k *KubeClient) LabelExistsInCluster(label string) error {
+	cmd := []string{"get", "pods", "--namespace=wego-system", "--show-labels"}
```

Comment:
comment for refactoring later, but this will be quite inefficient, as a cluster grows, querying for all pods takes quite some time, several seconds to minutes.

So, ideally, we should query for the apps with the label on them, something like:

```
 kubectl get app -l hash=my-hash
```